app/src/main/java/com/hadiyarajesh/notex/reminder/notification/NotificationBroadCastReceiver.kt

```diff
@@ -21,7 +22,7 @@ class NotificationBroadCastReceiver : HiltBroadcastReceiver() {
     }
 
     @Inject
-    lateinit var reminderWorkManager: ReminderWorkManager
+    lateinit var reminderWorkManager: MainNotificationWorker
```

Comment:
Always use correct naming for variables or at least it should be matched with Class Name to enhance code readability.